komorebic/src/main.rs

```diff
@@ -801,6 +801,9 @@ struct Stop {
     /// Stop masir if it is running as a background process
     #[clap(long)]
     masir: bool,
+    /// Do not restore windows after stopping komorebi
+    #[clap(long)]
```

Comment:
Let's add `#[clap(hide = true)]` as well to hide this from public docs as well

Reply:
👍 Done